source/configuration_manual/namespace/index.rst

```diff
@@ -126,11 +159,13 @@ See also :ref:`setting-namespace_hidden`.
 ``namespace/location``
 ----------------------
 
- - Default: :ref:`setting-mail_location`
- - Values: :ref:`string`
+- Default: :ref:`setting-mail_location`
+- Values:  :ref:`string`
 
 Specifies driver and parameters for physical mailbox storage.
```

Comment:
This could say also more clearly that it simply overrides the mail_location setting for this namespace.